Resources/Textures/Structures/Machines/VendingMachines/games.rsi/meta.json

```diff
@@ -0,0 +1,46 @@
+{
+    "version": 1,
+    "size": {
+        "x": 32,
+        "y": 32
+    },
+    "license": "CC-BY-SA-3.0",
+    "copyright": "Taken from tgstation https://github.com/tgstation/tgstation/commit/dd67e1e4e10bcf632b82b8b14b6a8479d80d5f78#diff-34173d670aca65c110ef18312c32abc4e05c84f33a9402248a32000e45acd0d3",
+    "states": [
+        {
+          "name": "normal-unshaded",
+          "delays": [
+            [
+              2.5,
+              0.1,
+              3.4,
+              0.1
+            ]
+          ]
+        },
+        {
+          "name": "broken"
+        },
+        {
+          "name": "normal"
+        },
+        {
+          "name": "eject-unshaded",
+          "delays": [
+            [
+              0.5,
+              0.1,
+              1.0,
+              0.1,
+              0.1
+            ]
+          ]
+        },
+        {
+          "name": "off"
+        },
+        {
+          "name": "panel"
+        }
+    ]
```

Comment:
Looks like games.png isn't used?
Looks like normal.png has duplicated texures with off.png and normal-unshaded so it can probably have parts cut out.

Reply:
Ok, was a bit confused there but I get it after looking at the dinneware vender.

I've removed games and normal, base is off.png which has normal layered on top.